docs/user/resources/04-50-gcp-vpc-peering.md

```diff
@@ -1,18 +1,27 @@
 # GcpVpcPeering Custom Resource
 
 The `gcpvpcpeering.cloud-resources.kyma-project.io` custom resource (CR) describes the VPC peering
-connection that you can create to allow communication between Kyma and a remote VPC in Google Cloud Platform (GCP).
+ that you can create to allow communication between Kyma and a remote VPC in Google Cloud Platform (GCP).
 It allows you to consume services available on the remote VPC from the Kyma cluster.
 
-### Required Permissions on the Remote Project <!-- {docsify-ignore} -->
-In order to create a VPC peering connection, the following permissions are required and must be granted to the Kyma service account provided:
+## Required Permissions on the Remote Project <!-- {docsify-ignore} -->
+In order to create a VPC peering, the following permissions are required and must be granted to the Kyma service account provided
+in the Service Account table below:
 
 | Parameter                          | Description                                                                 |
 |------------------------------------|-----------------------------------------------------------------------------|
 | compute.networks.addPeering        | Required to create the peering request in the remote project and VPC.       |
 | compute.networks.get               | Required to fetch the list of existing VPC peerings from the remote VPC.    |
```

Comment:
What is the difference between VPC and remote VPC? Is peering countable? I have not encountered that word in the plural form.